tests/zfs-tests/include/libtest.shlib

```diff
@@ -2286,6 +2286,8 @@ function add_user #<group_name> <user_name> <basedir>
 	fi
 
 	log_must useradd -g $gname -d $basedir/$uname -m $uname
+	echo "export PATH=\"$STF_PATH\"" >>$basedir/$uname/.profile
+	echo "export PATH=\"$STF_PATH\"" >>$basedir/$uname/.bash_profile
```

Comment:
Let's add .login too just in case the default shell on the system is tcsh or csh.